package/system/signify/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=signify
+PKG_VERSION:=29
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

Comment:
Wouldn't it be better to use `.tar.xz`? It should be smaller around 20 KB. But I haven't do compile test, but I was not able to download it from codeload.  Using "regular" Github works.

Reply:
How did you download it? I tried `https://codeload.github.com/aperezdc/signify/tar.xz/v29?/signify-29.tar.xz` but it 404.
What do you mean by regular download?